images/develop/docker-run.sh

```diff
@@ -480,6 +480,15 @@ if [[ ${DOLI_CRON} -eq 1 ]]; then
     exit 0
 fi
 
+if [[ "" != "${APACHE_REMOTEIP_CONF}" ]]; then
+    echo "RemoteIPHeader X-Forwarded-For" > /etc/apache2/mods-available/remoteip.conf
+    a2enmod remoteip
+fi
```

Comment:
okay that's maybe why It never worked out of the box for me
having `X-Client-IP` as an optional value for this line would be cool

Reply:
After this PR only if it checks that the file does not exist yet

Reply:
@williamdes  don't worry, I don't think this PR will get in, I felt that the consensus was against it

Reply:
I see..
Maybe check nextcloud?
I did not have the time to investigate why my docker logs have not got the remote ip